internal/compiler/generator/rust.rs

```diff
@@ -1046,7 +1046,7 @@ fn generate_sub_component(
         );
         quote!(#prop.get() as usize)
     } else {
-        quote!(core::usize::MAX)
+        quote!(usize::MAX)
```

Comment:
I'm a bit concerned about this one as this would break if someone had a made `struct usize`. But since we'd have the same problem two lines above, we can assume it is fine.

Reply:
That's a good point. Perhaps both should be replaced with the fully qualified `::core::primitive::usize`?